hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -222,6 +224,10 @@ AbfsThrottlingIntercept getIntercept() {
     return intercept;
   }
 
+  boolean shouldThrottleRetries() {
+    return throttleRetries;
+  }
+
```

Comment:
why is it in abfsClient. What risk is there to keep it in abfsRestOperation.

Reply:
Within AbfsRestOperation, would have to do client.getAbfsConfiguration.getShouldThrottleRetries. Would that be more preferable?

Reply:
Either of two is fine.
if we keep in abfsClient, it will be stored account level, and we dont need to check anything till abfsClient object is alived. In abfsRestOp, new field will be created as new object is created for each api call.
if we keep it in abfsRestOperation, it is something which is actually requried in abfsRestOperation. Though it doesn't matter. You may please resolve this comment.

Reply:
Right. That makes sense. We can directly invoke it from within AbfsRestOperation.